oms-active-directory-security-audit-solution/azuredeploy.json

```diff
@@ -0,0 +1,768 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "type": "string",
+          "metadata": {
+                "description": "Specify the workspace region"
+            }
+        },
+       
+        "workspaceRegionId": {
+            "type": "string",
+            "metadata": {
+                "description": "Specify the workspace region"
+            }
+        },
+
+        "pricingTier": {
+            "type": "string",
+            "defaultValue": "Standalone",
+            "allowedValues": [
+                "Free",
+                "Standalone",
+                "PerNode"
+            ],
+            "metadata": {
+                "description": "Service Tier: Free, Standalone, or PerNode"
+            }
+        }
+      
+    },
+    
+
+
+    "variables": {
+        "omsSolutions": {
+            "customSolution": {
+                "name": "AD Security Audit Solution",
+                "displayName": "AD Security Audit Solution",
+                "LogAnalyticsApiVersion": "2017-03-03-preview"
+            }
+        },
+        "LogAnalyticsApiVersion": "2017-03-03-preview",

```

Comment:
remove this you'll need a literal apiVersion

Reply:
done